### DISCUSSION TITLE
How to setup 2 external monitors?

### DISCUSSION THREAD

**Discussion by LordZeuss** [i3/i3#5957] [Q&A]
Hello, I was able to get i3 on my laptop configured right, where if I use xrandr I can get the two external displays to show. I have a few questions.

My laptop is 4k, so I have an Xresources file that has my dpi set to 150 so it looks good on my laptop screen. When connected to the monitors (2560x1080p ultra wides) the dpi is off due to the difference in aspect ratio. If I modify the Xresources file, it will only work on one screen (both externals are same monitor). How do I fix this? Can I set dpi per screen?

My other question is how can I set a screen as "main"? For example polybar shows on the wrong monitor. Is this a polybar or i3 setting?

**Comment by orestisfl** [comment]
In X11, dpi applies to the whole desktop. See https://wiki.archlinux.org/title/HiDPI#Multiple_displays for workarounds.

**Comment by okraits** [comment]
I would recommend to try if you can get the desired resolutions and scaling with xrandr.
To specify the main output with xrandr, use the `--primary` option.
